File: ClientApp/SuspendedStorefront/src/app/api/models/product-subscription.ts
/* tslint:disable */
/* eslint-disable */
import { Customer } from './customer';
import { Product } from './product';
export interface ProductSubscription {
  customer?: Customer;
  customerID?: string;
  fridayQuantity?: number;
  id?: string;
  isActive?: boolean;
  maxSuspendedQuantity?: number;
  mondayQuantity?: number;
  previousSubscriptionID?: string;
  product?: Product;
  productID?: string;
  recurringBasketID?: string;
  saturdayQuantity?: number;
  sundayQuantity?: number;
  thursdayQuantity?: number;
  tuesdayQuantity?: number;
  wednesdayQuantity?: number;
}

File: api/Models/ProductSubscription.cs
namespace SuspendedStorefront.Models
{
    public class ProductSubscription
    {
        public Guid ID { get; set; }
        public Guid RecurringBasketID { get; set; }

        public Guid ProductID { get; set; }

        public Guid CustomerID { get; set; }

        public bool IsActive { get; set; }

        public Guid PreviousSubscriptionID { get; set; }

        public int MondayQuantity { get; set; }
        public int TuesdayQuantity { get; set; }
        public int WednesdayQuantity { get; set; }
        public int ThursdayQuantity { get; set; }
        public int FridayQuantity { get; set; }
        public int SaturdayQuantity { get; set; }
        public int SundayQuantity { get; set; }

        public int MaxSuspendedQuantity { get; set; }

        public Customer Customer { get; set; }
        public Product Product { get; set; }


    }
}